composer.json

```diff
@@ -25,6 +26,7 @@
         "issues": "https://github.com/impress-org/givewp/issues"
     },
     "scripts": {
+        "test": "./vendor/bin/phpunit --bootstrap tests/src/bootstrap.php --no-configuration tests/src/",
```

Comment:
I forked the current implementation of unit testing, because there is some open discussion on cleaning up what otherwise appears to be an abandoned testing strategy. In the name of progress I started clean here and can refactor it back into a larger testing strategy.